OpenSim/Common/DataTable.h

```diff
@@ -0,0 +1,268 @@
+/* -------------------------------------------------------------------------- *
+ *                            OpenSim:  DataTable.h                           *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2015 Stanford University and the Authors                *
```

Comment:
I think the Authors field is there primarily so that people who want some credit can get it. I think using it should be optional and as Shrinidhi points out git will keep track of things for any serious blame purposes. I would prefer to see an empty `Authors:` line in the header block though so that it doesn't look forgotten.